lasy/utils/laser_utils.py

```diff
@@ -211,6 +206,78 @@ def get_full_field(laser, theta=0, slice=0, slice_axis="x", Nt=None):
     return env, ext
 
 
+def get_spectrum(grid, dim, bins=20, range=None, omega0=None, phase_unwrap_1d=None):
+    """
+    Get the the frequency spectrum of an envelope.
+
+    Parameters
+    ----------
+    grid : a Grid object.
+        It contains a ndarrays with the field data from which the
+        spectrum is computed, and the associated metadata. The last axis must
+        be the longitudinal dimension.
+        Can be the full electric field or the envelope.
```

Comment:
Can it really be either? The `get_frequency` call below contains `is_envelope=True`.

Reply:
Whether this supports both envelope and full field is something we could discuss. I believe both would be nice.

Reply:
I copy-pasted this docstring too fast. Currently it supports only envelope. I think we should make a general decision about this, so that all methods support both or only one representation.